prettier.config.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
   trailingComma: 'es5',
   printWidth: 80,
   semi: false,
-  jsxBracketSameLine: true,
```

Comment:
I sneaked this in. Annoyed me for a long time. How about we go closer to Prettier defaults?